client/app/components/Input/InputRadio.scss

```diff
@@ -0,0 +1,20 @@
+@import "../../styles/_global.scss";
+
+.wrapper {
+  display: flex;
+  flex-direction: row;
+
+  .content {
+    margin-right: 15px;
+  }
+
+  input {
+    margin-right: 5px;
+  }
+
+  label {
+    all: unset;
+    color: rgba(109, 8, 57, 0.9);
```

Comment:
You can replace this with `$mulberry-90` which comes from [here](https://github.com/ifmeorg/ifme/blob/master/app/assets/stylesheets/base/_colors.scss#L26).